types/sf_polygon_test.go

```diff
@@ -263,7 +263,8 @@ func TestSFPolygonMarshsalMapValue(t *testing.T) {
 	var data map[string]interface{}
 	var err error
 
-	wrapper = Wrapper{types.NewSFPolygonXY(testPolygonExternal, testPolygonInternal)}
+	wrapper = Wrapper{
+		types.NewSFPolygonXY(testPolygonExternal, testPolygonInternal)}
```

Comment:
I'm not gonna tell you to revert this change, but I don't get much value from this kind of reflowing. The fact that we're indenting with tabs makes the 80 column delimiter highly arbitrary, so... 'eh.

Reply:
Heh, fair enough. I don't mind reverting it.